src/context.ts

```diff
@@ -1,12 +1,17 @@
 import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
+import { createClient } from "@supabase/supabase-js";
+import { Database } from "./types/database";
+import { Logs } from "ubiquibot-logger";
+import { Env } from "./worker";
 
 export class Context<T extends WebhookEventName = WebhookEventName> {
   public key: WebhookEventName;
   public name: WebhookEventName;
   public id: string;
   public payload: WebhookEvent<T>["payload"];
+  public logger: Logs;
 
-  constructor(event: WebhookEvent<T>) {
+  constructor(event: WebhookEvent<T>, env: Env) {
```

Comment:
I have concerns around dealing with the env object because I know that cloudflare workers is finicky about this. 

@whilefoo rfc

Reply:
@pavlovcik thank you for the hint, I will refactor it.

Reply:
I'd pass the logger as a parameter instead of creating it inside